recipe/2024-01-global-scope-secret-store.md

```diff
@@ -0,0 +1,261 @@
+# Adding support to extend the capabilities of secretStores to a global scope.
+
+* **Status**: Pending
+* **Author**: Vishwanath Hiremath (@vishwahiremat)
+
+## Overview
+
+Today we have Application.Core/secretStores resource to securely manage secrets for the Application. However a gap exists if user wants to create a secret before application or environment is created. To address this limitation, we need to enhance the capabilities of the existing secretStores resource. The objective is to extend its support to a global scope, enabling users to store secrets prior to the creation of the application or environment.
+
+
+## Terms and definitions
+| Term     | Definition                                                                                                                                                                                                 |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Global scope | The ability of the secretStores resource to operate universally, allowing users to manage secrets independently of a particular application or environment. |
+| secretStores | Resource used to securely manage secrets for Environment and Application. |
+| Private Terraform Repository | A private Terraform repository typically refers to a version control repository that contains Terraform module code, but is not publicly accessible. | 
+
+## Objectives
+
+> **Issue Reference:** https://github.com/radius-project/radius/issues/7030
+
+### Goals
+
+- Enable support to extend the capabilities of secretStores to a global scope.
+
+### Non goals
```

Comment:
@vishwahiremat is https://github.com/radius-project/design-notes/pull/38/files#r1466964211 a non goal for the scope of this document? If so, let's please add it here with the reasoning why.

Reply:
thats correct, its a non goal for this task. Will add it as part of the doc